Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -1022,7 +1022,7 @@ private extension PurchasesOrchestrator {
         }
     }
 
-    func syncPurchases(receiptRefreshPolicy: ReceiptRefreshPolicy,
+    func syncPurchases(receiptRefreshAllowed: Bool,
```

Comment:
Since `ReceiptRefreshPolicy` is very tied to SK1, I changed the parameter to a more SK-agnostic one and decide in the method body what policy to use for each SK version.